docs/bgp.md

```diff
@@ -0,0 +1,193 @@
+# Configuration of BGP peering
+
+By default, when running Calico on Docker, a full BGP mesh is used to
+distribute routes between all of the Calico nodes.  This approach has scale
+limitations, and alternative topologies would be recommended for building a
+large scale network.
+
+This document describes the commands available in calicoctl for managing BGP.
+
+
+### Configuring the default node AS number
+
+When creating a Calico node, you can optionally specify an AS number to use for
+the node.  If no AS number if specified, the node will use the global default
+value.
+
+Use the `calicoctl bgp default-node-as` command to set the global default AS
+number.  If no value is configured, the default AS number is 64511.
+
+If all of your Calico nodes are within the same AS, but you require a
+different AS number to be used (e.g because you are peering with a border
+router), changing the default AS number to the value you require eliminates
+the need to explicitly set it on a per Calico node basis.  For more
+complicated toplogies where you are explicitly setting the AS number on each
+node, the default value will not be used and therefore using this command is
+not necessary.
+
+#### Example
+
+To set the default AS number to 64512, run the following calicoctl command on
+any node:
+
+	$ calicoctl bgp default-node-as 64512
+	
+To view the current default value, run the command without specifying an AS
+number, the command will output the current value.
+
+	$ calicoctl bgp default-node-as
+	64512
+	
+
+### Disabling the full node-to-node BGP mesh
+
+If you are explicitly configuring the BGP topology for your Calico network, 
+you may wish to disable the full node-to-node mesh.  Use the
+`calicoctl bgp node-mesh` command to disable or re-enable the mesh.
+
+If you are building your network from scratch and do not need the full
+node-to-node mesh we recommend turning off the mesh before configuring your
+nodes.  If you are updating your network from a full-mesh topology to a
+different topology (e.g. to start using a cluster of route reflectors to
+increase scaling), configure the appropriate peers before disabling the mesh
+to ensure continuity of service.
```

Comment:
So the node-to-node mesh is in additional to any explicitly configured peerings? It might be worth making that more explicit in the doc?